vidify/gui/api/spotify_web.py

```diff
@@ -4,9 +4,9 @@
 
 import logging
 
-from PySide2.QtWidgets import (QWidget, QLabel, QPushButton, QHBoxLayout,
+from qtpy.QtWidgets import (QWidget, QLabel, QPushButton, QHBoxLayout,
                                QVBoxLayout)
```

Comment:
./vidify/gui/api/spotify_web.py:8:32: E127 continuation line over-indented for visual indent

you need to remove a couple spaces to fix the spacing